cmd/kube-apiserver/app/server.go

```diff
@@ -568,7 +570,9 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 	}
 
 	if s.Etcd.EncryptionProviderConfigFilepath != "" {
-		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(s.Etcd.EncryptionProviderConfigFilepath)
+		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(
```

Comment:
I think you should structure this as a totally separate library that returns the correct interface, rather than modifying the existing library.